test/csv/parse/test_quote_char_nil.rb

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# frozen_string_literal: false
+
+require_relative "../helper"
+
+class TestCSVParseQuoteCharNil < Test::Unit::TestCase
+  extend DifferentOFS
+
+  def test_general
+    [[%Q{a,b},  ["a", "b"]],
+     [%Q{a,,,}, ["a", nil, nil, nil]],
+     [%Q{,},    [nil, nil]],
+    ].each do |edge_case|
+      assert_equal(edge_case.last, CSV.parse_line(edge_case.first, quote_char: nil))
+    end
+  end
+
+  def test_unquoted_value_multiple_characters_col_sep
+    data = %q{a<b<=>x}
+    assert_equal([[%Q{a<b}, "x"]], CSV.parse(data, col_sep: "<=>", quote_char: nil))
+  end
+
+  def test_csv_header_string
+    # activate headers
+    csv = nil
+    data = <<~DATA
+      first,second,third
+      A,B,C
+      1,2,3
+    DATA
+    assert_nothing_raised(Exception) do
+      csv = CSV.parse(data, headers: "my,new,headers", quote_char: nil)
+    end
```

Comment:
Could you remove `assert_nothing_raised`?
The assertion is meaningless.

Reply:
We can use one `assert_equal` in this case:

```ruby
    assert_equal(CSV::Table.new(...),
                 CSV.parse(data, headers: "my,new,headers", quote_char: nil))
```

Reply:
Fixed. https://github.com/ruby/csv/pull/79/commits/efa8916848b63b8013bc415ed147ce66247e5068